src/action.ts

```diff
@@ -18,9 +18,15 @@ const COVERAGE_HEADER = ":loop: **Code coverage**\n\n"
 export async function run() {
   let workingDirectory = core.getInput("working-directory", { required: false })
   let cwd = workingDirectory ? resolve(workingDirectory) : process.cwd()
-  const CWD = cwd + sep
-  const RESULTS_FILE = join(CWD, "jest.results.json")
+  let resultFileName = core.getInput("results-file", { required: false })
+  if (resultFileName === "") {
+    resultFileName = "jest.results.json";
+  }
```

Comment:
This can be one line

```suggestion
  const resultFileName = core.getInput("results-file", { required: false }) || "jest.results.json"
```